datacats/environment.py

```diff
@@ -23,7 +23,8 @@
 from datacats.template import ckan_extension_template
 from datacats.scripts import (WEB, SHELL, PASTER, PASTER_CD, PURGE,
     RUN_AS_USER, INSTALL_REQS, CLEAN_VIRTUALENV, INSTALL_PACKAGE,
-    COMPILE_LESS, DATAPUSHER, INSTALL_POSTGIS, ADJUST_DEVINI)
+    COMPILE_LESS, DATAPUSHER, INSTALL_POSTGIS, ADJUST_DEVINI,
```

Comment:
Yes, this has been bothering me for a while but not enough to fix it.

I'd merge a PR that changes these to `from datacats import scripts` and uses `scripts.FOO` instead